docs/source/tutorial/package_testing.rst

```diff
@@ -39,23 +41,46 @@ Near the end of the log, you should see something like this:
     2019-05-01 13:08:40,563 - DataIngestPipeline - INFO - ❌ Count Analysis Failed!
     See /path/to/my_study/logs/counts_for_ingest.log for details
 
-Inside of that ``counts_for_ingest.log`` file, you should some things that look
+Inside of that ``counts_for_ingest.log`` file, you should something that look
```

Comment:
I should what? Also looks.

Reply:
Oops, `you should see something that looks like this`